code/modules/mob/living/carbon/xenomorph/life.dm

```diff
@@ -98,9 +98,9 @@
 /mob/living/carbon/xenomorph/proc/handle_critical_health_updates()
 	var/turf/T = loc
 	if((istype(T) && locate(/obj/effect/alien/weeds) in T))
-		heal_wounds(XENO_RESTING_HEAL + (warding_aura * 0.5) * 0.5) //Warding pheromones provides 0.125 HP per second per step, up to 1.25 HP per tick.
+		heal_wounds(XENO_RESTING_HEAL * 0.5)
```

Comment:
Intentional that you're also halving the base heal value on top of not applying a warding bonus?

Reply:
splinter just doesn't know how to do math lel

Reply:
> 
> 
> Intentional that you're also halving the base heal value on top of not applying a warding bonus?

it halved the base heal + warding aura before

Reply:
Nah, before it had the base resting heal plus a quarter of the warding aura. B + (W * .5) * .5